src/prettytable.cpp

```diff
@@ -159,7 +193,28 @@ PrettyTable::get_icon_from_default(const ProcInfo &info)
     return pix;
 }
 
+Glib::RefPtr<Gdk::Pixbuf>
+PrettyTable::get_icon_from_gio(const ProcInfo &info)
+{
+    gchar **cmdline = g_strsplit(info.name, " ", 2);
+    const gchar *executable = cmdline[0];
+    Glib::RefPtr<Gdk::Pixbuf> icon;
+
+    if (executable) {
+        Glib::RefPtr<Gio::AppInfo> app = this->gio_apps[executable];
+        Glib::RefPtr<Gio::Icon> gicon;
+        Gtk::IconInfo info;
```

Comment:
Build warning.
```
prettytable.cpp: In member function 'Glib::RefPtr<Gdk::Pixbuf> PrettyTable::get_icon_from_gio(const ProcInfo&)':

prettytable.cpp:206:23: warning: declaration of 'Gtk::IconInfo info' shadows a parameter [-Wshadow]

  206 |         Gtk::IconInfo info;

      |                       ^~~~

prettytable.cpp:197:48: note: shadowed declaration is here

  197 | PrettyTable::get_icon_from_gio(const ProcInfo &info)

      |                                ~~~~~~~~~~~~~~~~^~~~
```
https://travis-ci.org/github/mate-desktop/mate-system-monitor/jobs/711714681#L2266

Reply:
The variable `Gtk::IconInfo info;` has been removed as it's not used in the scope.